tICA/tICAPipeline.sh

```diff
@@ -149,6 +160,25 @@ then
     signalTxtName="ReCleanSignal.txt"
 fi
 
+if [[ "$IsLongitudinal" == "1" ]]; then
+    if [[ "$ICAmode" != "REUSE_ICA" ]]; then
+        log_Err "mode other than REUSE_ICA is not supported in longitudinal processing"
+    fi
+    if [[ "$TemplateLong" == "" || "$Subject" == "" || "$extractNameOut" == "" ]]; then 
+        log_Err_Abort "--extract-fmri-out, --longitudinal-template and --longitudinal-subject are required in longitudinal mode."
+    fi
+    if [[ "$ExtractAllRunsLong" == "1" ]]; then 
```

Comment:
For style, I prefer handling booleans like this (this is why opts_StringToBool normalizes to a number instead of something like "true").
```suggestion
    if ((ExtractAllRunsLong)); then 
```

Reply:
Ok